From 514c1438440c21f7f9a6e2f36ee81fd86bc77955 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:52:15 -0400
Subject: [tools] include: Include <linux/compiler.h> from asm/bug.h

Message-id: <1408953210-26343-248-git-send-email-jolsa@redhat.com>
Patchwork-id: 88956
O-Subject: [PATCH RHEL7.1 BZ1133083 247/322] tools include: Include <linux/compiler.h> from asm/bug.h
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 741a0c59032faeddac80a6237f3d7846231a3740
Author: Namhyung Kim <namhyung@kernel.org>
Date: Thu Jan 9 23:00:55 2014 +0900

description
===========
Since it uses unlikely() macro inside WARN()
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/include/asm/bug.h b/tools/include/asm/bug.h
index eca78df..9e5f484 100644
--- a/tools/include/asm/bug.h
+++ b/tools/include/asm/bug.h
@@ -1,6 +1,8 @@
 #ifndef _TOOLS_ASM_BUG_H
 #define _TOOLS_ASM_BUG_H
 
+#include <linux/compiler.h>
+
 #define __WARN_printf(arg...) do { fprintf(stderr, arg); } while (0)
 
 #define WARN(condition, format...) ({  \
-- 
1.7.1